docs/source/markdown/podman-secret-create.1.md

```diff
@@ -36,6 +36,10 @@ Read secret data from environment variable
 
 Print usage statement.
 
+#### **--label**, **-l**=*key=val1,key2=val2*
+
+Add label to secret
```

Comment:
```suggestion
Add label to secret.
```

Can we elaborate a bit on what these labels will do or can be used for. Will they be display in `secret ls/inspect` etc?